docs/examples/proxy/README.md

```diff
@@ -0,0 +1,65 @@
+# Elasticsearch proxy example
+
+This folder contains an example of how you could build a lightweight proxy
+that you can put between your frontend code and Elasticsearch if you don't
+have a more sophisticated backend in place yet.
+
+> **IMPORTANT:** this is not a production ready code & purely for demonstration purposes,
+> we make no guarantees on it's security and stability
+
+This project is designed to be deployed on [Vercel](https://vercel.com/), a cloud platform
+for static sites and Serverless Functions. You can use also other functions providers,
+such as [Google Cloud functions](https://cloud.google.com/functions).
+
+## Project structure
+
+The project already comes with three endpoints:
+
+- `/api/search`: run a search, it requires the `'read'` permissions
+- `/api/autocomplete`: run a autocomplete suggestion, it requires the `'read'` permissions
+- `/api/index`:  index or updates a document, it requires the `'write'` permissions
+- `/api/delete`  delete a document, it requires the `'write'` permissions
+
+Inside `utils/authorize.js` you can find the authorization logic for the endpoints.
+In each endpoint you should configure the `INDEX` variable.
+
+## How to use
+
+First of all, create an account on Vercel and create a deployment on Elastic Cloud, if you
+don't have an account on Elastic Cloud, you can create one with a free 14-day trial
+of the [Elasticsearch Service](https://www.elastic.co/elasticsearch/service).
+
+### Configure Elasticsearch
+
+Once you have created a deployment on Elastic Cloud copy the generated Cloud Id and the credentials.
+Then open `utils/prepare-elasticsearch.js` and fill your credentials. The script will generate
```

Comment:
Nit. We try to use present simple tense in the docs whenever it's possible as it is the easiest to read and skim.
```suggestion
Then open `utils/prepare-elasticsearch.js` and fill your credentials. The script generates
```